app/src/main/java/com/kickstarter/ui/fragments/PledgeFragment.kt

```diff
@@ -192,6 +191,14 @@ class PledgeFragment : BaseFragment<PledgeFragmentViewModel.ViewModel>(), Reward
                 .compose(observeForUI())
                 .subscribe { (cards_recycler.adapter as RewardCardAdapter).takeCards(it) }
 
+        this.viewModel.outputs.card()
```

Comment:
Since the `PledgeFragment` is a fairly complex fragment, I would choose a more descriptive name for the output to make it easier to understand what is this output. (ie. `selectedCard`, `addedCard`...)